bin/proving-service/src/commands/mod.rs

```diff
@@ -1,87 +1,60 @@
 use clap::Parser;
-use figment::{
-    providers::{Format, Toml},
-    Figment,
-};
-use init::Init;
 use proxy::StartProxy;
-use serde::{Deserialize, Serialize};
 use tracing::instrument;
 use update_workers::{AddWorkers, RemoveWorkers, UpdateWorkers};
 use worker::StartWorker;
 
-use crate::utils::{MIDEN_PROVING_SERVICE, PROVING_SERVICE_CONFIG_FILE_NAME};
+use crate::utils::MIDEN_PROVING_SERVICE;
 
-pub mod init;
 pub mod proxy;
 pub mod update_workers;
 pub mod worker;
 
-/// Configuration of the proxy.
-///
-/// It is stored in a TOML file, which will be created by the `init` command.
-/// It allows manual modification of the configuration file.
-#[derive(Debug, Serialize, Deserialize)]
-pub struct ProxyConfig {
-    /// Host of the proxy.
-    pub host: String,
-    /// Port of the proxy.
-    pub port: u16,
-    /// Worker update service port.
-    pub workers_update_port: u16,
-    /// Maximum time in seconds to complete the entire request.
-    pub timeout_secs: u64,
+#[derive(Debug, Parser)]
+pub(crate) struct ProxyConfig {
+    /// Time in milliseconds to poll available workers.
```

Comment:
nit: Considering the health check setting is called `health_check_interval_secs`, I'd suggest renaming this variable to use a similar convention. Could be something like `available_workers_polling_interval_ms` or maybe `availability_check_interval_ms`.

Also, this setting is for polling for availability and assigning tasks, right? If so it would be good to specify int he doc comment:
```suggestion
/// Interval in milliseconds at which the system polls for available workers to assign new tasks.
```

Reply:
Sure, I just update the var to `available_workers_polling_interval_ms`.